docs/demos/milestone-1/fendermint-demo.sh

```diff
@@ -25,7 +25,7 @@ cat test-network/keys/alice.pk
 fendermint \
   genesis --genesis-file test-network/genesis.json \
   add-account --public-key test-network/keys/alice.pk \
-              --balance 1000000000000000000
+              --balance 1000
 
```

Comment:
I'm not sure I would alter this demo script to be honest. The README in the milestone directory says these commands reflect the state as it was back then, although I was too lazy to include a commit, but it should point for example to 7f4c155f35b374327a33c7fccec6df0e71870f07 which is one before the switch to CometBFT.

I wanted this script to be as it appeared in the video, a snapshot at Milestone 1, for reference, and only keep the `running.md` file up to date.